### ISSUE TITLE
[Android][WSL2] Compile android on wsl2 with 20.04 get error.

### ISSUE THREAD

**Issue by talregev** [BOINC/boinc#3792]
> **Describe the bug**
> A clear and concise description of what the bug is.

When compile android on 20.04 in wsl2 it get error.

> **Steps To Reproduce**

1. Setup wsl2 with ubuntu 20.04
2. `sudo apt install git python make m4 pkg-config dh-autoreconf libmysqlclient-dev libssl-dev python3-mysqldb libnotify-dev`
3. git clone https://github.com/BOINC/boinc/
4. run android scrip: `./build_all.sh`

> **Expected behavior**
> A clear and concise description of what you expected to happen.

Compile without error.

**Screenshots**

**System Information**
 - OS: Windows 10, wsl2
 - BOINC Version: master.

> **Additional context**
> Add any other context about the problem here.

```bash
1 warning generated.
  CXXLD    boinc_client
clang70++: error: no such file or directory: 'libwhetneon.a'
clang70++: error: no such file or directory: 'libwhetvfp.a'
make[2]: *** [boinc_client] Error 1
make[1]: *** [all-recursive] Error 1
make: *** [all] Error 2
```

I also compile on ubuntu 18.04 with wsl2. There isn't issue there. It compile all arm versions.
In ubuntu 20.04, after the error I re-order the arm versions. let arm be the last one. Also delete the buildCache folder. It fail on the arm compile, with the same error.

**Comment by truboxl** [comment]
https://github.com/BOINC/boinc/pull/3717

**Comment by talregev** [comment]
@truboxl Why it not merge to the master?

**Comment by truboxl** [comment]
@talregev Good question. No idea.

**Comment by talregev** [comment]
@truboxl Who is responsible to review this? I will try to talk with him.

**Comment by AenBleidd** [comment]
I'll review it

**Comment by talregev** [comment]
@AenBleidd I can confirm this fix compile on wsl2 ubuntu 20.04

**Comment by talregev** [comment]
Issue fix. Thanks @truboxl @AenBleidd
